README.rst

```diff
@@ -223,6 +181,68 @@ We can now run Yadage locally as follows:
 Let us check whether the resulting plot is the same as the one showed in the
 documentation:
 
+.. code-block:: console
+
+    $ diff outputs/plot.png  ./docs/plot.png
+
+Local testing with CWL
+=========================
+
+Let us test whether the CWL workflow execution works locally as well.
```

Comment:
The example works nicely for me :+1: 

One documentation comment to simplify this part; please see https://github.com/reanahub/reana-demo-helloworld/pull/17/files#r170864827

Reply:
FYI I'm using a setup like the following to test `master` branches of components in PROD-like conditions:

(1) Bring up fresh new Minikube:

```console
$ minikube start --kubernetes-version="v1.6.4" --vm-driver=kvm2
$ eval "$(minikube docker-env)"
```

(2) Compile LATEST versions of all REANA cluster components: (with `master` branch or with whatever branch I have checked out that want to test)

```console
$ cd src
$ for component in reana-job-controller reana-message-broker reana-server reana-workflow-controller reana-workflow-engine-yadage reana-workflow-engine-cwl reana-workflow-monitor; do \
     cd $component; \
     docker build -t reanahub/$component .;\
     cd ..; \
  done
```
(3) Bring up PROD-like cluster with LATEST component images:

```console
$ cd src/reana-cluster
$ cp reana_cluster/configurations/reana-cluster.yaml .
$ sed -i -e 's,0.1.0,latest,g' reana-cluster.yaml 
$ reana-cluster -f reana-cluster.yaml init
```

P.S. I have some helper scripts like `reana-git-upgrade`, `reana-docker-build` and friends that automatise this process (and the push t Docker Hub).  Can clean and commit one day...